cypress/commons/actions/generic/TableParameters.js

```diff
@@ -142,6 +150,10 @@ function setFileExportName(fileName) {
   getExportFileNameInput().type('{selectAll}{backspace}' + fileName + '{enter}');
 }
 
+function triggerFullscreenButton(tableParameterElement) {
```

Comment:
Could we rename this function to `toggleFullscreen`?

Reply:
DONE